x-pack/plugins/ml/common/types/anomaly_detection_jobs/job.ts

```diff
@@ -85,3 +85,5 @@ export interface CustomRule {
   scope: object;
   conditions: object;
 }
+
+export { JobStat, JobExistResult } from '../../../server/models/data_recognizer/data_recognizer';
```

Comment:
These types do not belong in this file as they are not related to the anomaly detection job configuration.
I'd suggest moving them to a `data_recognizer` file in the `common/types` folder.

Reply:
i have moved those interfaces to data_recognizer

Reply:
i see you've moved all interfaces from `data_recognizer` to common. i don't think that was needed, as you only need access to `JobStat` and `JobExistResult`
Apologies if my comment was unclear, I only meant for you to move the ones you need.

Reply:
I understood you correctly, but i felt since i am creating a specific type file, it would make sense to move all those interfaces to same file. That's also consistent with other files in the application.

Reply:
We try not to expose interfaces that are internal. Most of the interfaces that have moved are only needed in the file they were in.
I also realised after making my original comment that we already have a file that these two interfaces should go in. 
`ml/common/types/modules.ts`
It was my intention to merge the two files after your PR was in.

Reply:
got it, will reverse it and will only move which i need for this PR.

Reply:
done that, thanks for keeping a check :)